blueman/gui/manager/ManagerDeviceMenu.py

```diff
@@ -213,6 +252,32 @@ def Generate(self):
             self.append(item)
             return
 
+        # Generic (dis)connect
+        # LE devices do not appear to expose certain properties like uuids until connect to at least once.
+        # show generic connect for these as we will not show any way to connect otherwise.
+        device_uuids = self.SelectedDevice['UUIDs']
+        show_generic_connect = not device_uuids
+        for uuid in device_uuids:
+            if ServiceUUID(uuid).short_uuid in (
+                AUDIO_SOURCE_SVCLASS_ID, AUDIO_SINK_SVCLASS_ID, GENERIC_ACCESS_SVCLASS_ID,
+                HANDSFREE_AGW_SVCLASS_ID, HEADSET_SVCLASS_ID, HID_SVCLASS_ID
```

Comment:
I'm sorry but I have to throw in yet another discovery. 😆

It probably does not make sense to provide this for *every* LE device but only if there are auto-connectable services.

See https://github.com/blueman-project/blueman/pull/575#issuecomment-270059859 on those. I think we're still missing `HANDSFREE_SVCLASS_ID` here for this entry in `struct default_settings`:

```
           {
                .uuid           = HFP_AG_UUID,
                .name           = "Hands-Free Voice gateway",
                .priority       = BTD_PROFILE_PRIORITY_HIGH,
                .remote_uuid    = HFP_HS_UUID,
                .channel        = HFP_AG_DEFAULT_CHANNEL,
                .authorize      = true,
                .auto_connect   = true,
                .get_record     = get_hfp_ag_record,
                .version        = 0x0105,
        }
```
Beside audio sink and source and HID `auto_connect` is now set in `struct btd_profile hog_profile` and `struct btd_profile midi_profile`.

The short UUID for `struct hog_profile` is 0x1812 (`HOG_UUID16` in the BlueZ code). I did not find this is my initial analysis as it used a different auto-connect method before BlueZ 5.43, see [this commit](https://git.kernel.org/pub/scm/bluetooth/bluez.git/commit/profiles/input/hog.c?id=7d9718cfcc11eaa9d8059e721301cdc00ef8c82e). I checked that there are no other profiles that use that anymore.

The `MIDI_UUID` is `03B80E5A-EDE8-4B33-A751-6CE34EC4C700` and as it's no reserved UUID there is no 16-bit version. The profile was added in BlueZ 5.44. I see three possible solutions for this:
- Ignore it completely. This means that somebody with BlueZ >= 5.44 will not see the generic connect entry for a device with a MIDI service as the only auto-connecting service.
- Enable generic connect for it. This means that somebody with BlueZ < 5.44 will see the generic connect entry for a device with a MIDI service but no (other) auto-connecting service but nothing will get connected when he uses it.
- Enable generic connect for it depending on the version of BlueZ.